### PR TITLE
[ADD] Client to Calendar Event

### PR DESCRIPTION
**Pull request by eilst**

### CODE REVIEW

File: project_event/tests/test_project_event_reservation.py [hunk 1]
```diff
@@ -147,3 +147,16 @@ def test_020_event_action_accept(self):
...
+        wiz = self.env['reservation.validation.wiz'].browse(res['res_id'])
+        wiz.confirm_accept_reservation()
+        event_ids = self.project_1.task_ids.mapped(
+            'child_ids').mapped('reservation_event_id')
+        for event_id in event_ids:
+            reservation_event = self.env['calendar.event'].browse(
+                event_id)
+            self.assertEqual(
+                reservation_event.partner_id.id,
+                self.project_1.partner_id.id)
```
Comment (rimbendhaou) [inline project_event/tests/test_project_event_reservation.py]: return to line (PEP8)

File: project_event/tests/test_project_event_reservation.py [hunk 2]
```diff
@@ -147,3 +147,16 @@ def test_020_event_action_accept(self):
     def test_050_action_draft(self):
         self.project_1.action_draft()
         self.assertEqual(self.project_1.state, 'draft')
+
+    def test_60_check_partner_id_calendar_event_as_client_of_task(self):
```
Comment (rimbendhaou) [inline project_event/tests/test_project_event_reservation.py]: test_060...

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/tests/test_project_event_reservation.py (hunk 2 above)
Comment (eilst) [reply project_event/tests/test_project_event_reservation.py]: Done

File: project_event/tests/test_project_event_reservation.py (hunk 1 above)
Comment (eilst) [reply project_event/tests/test_project_event_reservation.py]: Done

**Review by rimbendhaou — APPROVED** [review]

**Pull request merged**